utils/dataset_manifest/core.py

```diff
@@ -707,27 +720,27 @@ def validators(self):
     @staticmethod
     def _validate_first_item(_dict):
         if not isinstance(_dict['name'], str):
-            raise ValueError('Incorrect name field')
+            raise InvalidFieldValueError('Incorrect name field')
         if not isinstance(_dict['extension'], str):
-            raise ValueError('Incorrect extension field')
+            raise InvalidFieldValueError('Incorrect extension field')
         # FIXME
         # Width and height are required for 2D data, but
         # for 3D these parameters are not saved now.
         # It is necessary to uncomment these restrictions when manual preparation for 3D data is implemented.
 
         # if not isinstance(_dict['width'], int):
-        #     raise ValueError('Incorrect width field')
+        #     raise InvalidFieldValueError('Incorrect width field')
         # if not isinstance(_dict['height'], int):
-        #     raise ValueError('Incorrect height field')
+        #     raise InvalidFieldValueError('Incorrect height field')
 
 def is_manifest(full_manifest_path):
-    return _is_video_manifest(full_manifest_path) or \
-        _is_dataset_manifest(full_manifest_path)
+    return is_video_manifest(full_manifest_path) or \
+        is_dataset_manifest(full_manifest_path)
 
-def _is_video_manifest(full_manifest_path):
+def is_video_manifest(full_manifest_path):
```

Comment:
This seems like an unnecessary change. `_is_video_manifest` isn't used from other modules, so it doesn't need to be public.

Reply:
It can be used in the server, which uses this module as a package.

Reply:
But _is_ it used in the server? Doesn't seem like it.

Reply:
2 other functions (`is_dataset_manifest` and `is_manifest`) are used now. While it is not used currently, it looks natural to have them all public.